### ISSUE TITLE
Not getting updates since 1.34.0 on Debian10/KDE without any errors

### ISSUE THREAD

**Issue by mYnDstrEAm** [VSCodium/vscodium#600]
**Describe the bug**
I'm not getting any updates since version since v 1.34.0 on Debian10/KDE. I implemented [this](https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/-/issues/36#note_395793123) after getting the "429 Too Many Requests" errors earlier and just redid that.

`sudo apt-get update` returns:

Get:1 https://paulcarroty.gitlab.io/vscodium-deb-rpm-repo/debs vscodium InRelease [X,XXX B]

`sudo apt-get upgrade` returns:

```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Calculating upgrade... Done
The following packages have been kept back:
  python3-psycopg2
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
```

**Please confirm that this problem is VSCodium-specific**
- [ ] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.
(Unrelated to MS VS Code)

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
1. Use/install Debian10/KDE
2. Add the repo and install VsCodium
3. Use VsCodium and then try to update it with apt-get

**Expected behavior**
I expected to get updates for 1.52.1

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Debian10
 - Architecture amd64
 - Version 1.34.0

**Additional context**

**Comment by mYnDstrEAm** [comment]
Moved it [here](https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo/-/issues/50)

**Comment by mYnDstrEAm** [comment]
The python3-psycopg2 message is unrelated to this issue and resolved now. I stopped getting updates since 1.34 and didn't get any notification about it (like after running `sudo apt update`) on multiple machines. Why is that? I thought I got a vscodium update months ago but as I had 1.34 installed that must be wrong...I think I got the 429 problem for months and implemented the workaround not long after it was posted.
